forc-plugins/forc-fmt/src/main.rs

```diff
@@ -154,7 +154,11 @@ fn format_file(
                 // TODO: Support formatting for incomplete/invalid sway code.
                 // https://github.com/FuelLabs/sway/issues/5012
                 debug!("{}", err);
-                bail!("Failed to compile: {:?}", file);
+                if let Some(file) = file.to_str() {
+                    bail!("Failed to compile {}\n{}", file, err);
+                } else {
+                    bail!("Failed to compile.\n{}", err);
+                }
```

Comment:
Was this change necessary? Seems the previous code achieves the same outcome in fewer lines.

Reply:
@eureka-cpu They are not the same thing. The error was not being printed previously, only logged. Right now the error is part of the output. In fact, adding the error helped me fix the issue with `hash.sw`.

Converting error from 

```
error: Formatting skipped due to error.
error: Failed to compile: "/Users/cr-fuel/projects/fuel/sway/sway-lib-std/src/hash.sw"
```

To 

```
error: Formatting skipped due to error.
error: Failed to compile /Users/cr-fuel/projects/fuel/sway/sway-lib-std/src/hash.sw
Cannot format raw hashbang attribute. If this is intended to be a doc comment try using the `//!` syntax instead
```

Reply:
I see, also I wrote that error message haha

Nice to see it there